### PR TITLE
Moved ittapi to thirdparty

### PR DESCRIPTION
**Pull request by ilyachur**

### CODE REVIEW

File: thirdparty/CMakeLists.txt [hunk 1]
```diff
@@ -14,4 +14,35 @@
 # limitations under the License.
 # ******************************************************************************
 
+if(ENABLE_PROFILING_ITT)
```
Comment (ilya-lavrenov) [inline thirdparty/CMakeLists.txt:17]: maybe a separate directory `thirdparty/ittnotify/CMakeLists.txt`?

Comment (ilyachur) [reply thirdparty/CMakeLists.txt:17]: Do you mean  to. add ittnotify as a submodule or just create a folder with cmake script?

Comment (ilya-lavrenov) [reply thirdparty/CMakeLists.txt:17]: > just create a folder with cmake script?

yes. `thirdparty` just a root cmake where only `add_subdirectory` calls.

Comment (ilyachur) [reply thirdparty/CMakeLists.txt:17]: Ok, fixed

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**